storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1854,6 +1854,17 @@ class Rdb_transaction {
   bool m_is_delayed_snapshot = false;
   bool m_is_two_phase = false;
 
+private:
+  /* Number of RockDB savepoints taken */
+  int m_n_savepoints;
```

Comment:
m_n_savepoints and m_writes_at_last_savepoint need to be initialized at similar locations to m_write_count, no?

Reply:
Yes. Lost that initialization when re-working the patch. Will now add it back